Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1390,6 +1446,26 @@ def update_remote_system_command(args, params, service: client.Service, auth_tok
     else:
         demisto.debug(f'Incident corresponding to notable {notable_id} was not changed.')
 
+    if entries:
+        for entry in entries:
+            entry_tags = entry.get('tags', [])
+            demisto.debug(f'Got the entry tags: {entry_tags}')
+            if comment_tag_to_splunk in entry_tags:
+                demisto.debug('Add new comment')
+                comment_body = f'{entry.get("contents", "")}\n {COMMENT_MIRRORED_FROM_XSOAR}'
```

Comment:
Why to do it deperatly ?
there is call to `update_notable_events` in line 1427can we combined them (and what is the `comment=changed_data['comment']` in line 1428?)